fix(copy-button): fix type of feedback text

asudoh opened the pull request:
Fixes #4606.

#### Changelog

**Changed**

- Choice of type token for feedback text.

#### Testing / Reviewing

Testing should make sure copy button and code snippet is not broken.

asudoh committed:
fix(copy-button): fix type of feedback text

Fixes #4606.

emyarod reviewed (commented):
copy button and inline code snippet look correct, but single line and multiline code snippet feedback text still appear to be using a different type token

![image](https://user-images.githubusercontent.com/8265238/68865811-0c1a7b80-06b9-11ea-9371-c31fca26a781.png)

![image](https://user-images.githubusercontent.com/8265238/68865837-16d51080-06b9-11ea-9d47-da9e51d9bc70.png)

abbeyhrt commented:
I'm seeing the same thing as @emyarod

laurenmrice reviewed (changes requested):
Vanilla and React:

-The `code snippet single line` and the `copy button` should 4px between the border of the hover and the tip of the fin of the tooltip. Right now its too close. 
![Screen Shot 2019-11-14 at 1 22 25 PM](https://user-images.githubusercontent.com/43969356/68889409-7b589580-06e2-11ea-88f2-75c2e0e26486.png)

-Seeing the same thing as andrew and abbey about the type

joshblack reviewed (commented):
Seems great with feedback from above 😄

asudoh committed:
fix(code-snippet): fix tooltip offset

joshblack commented on packages/components/src/components/code-snippet/_code-snippet.scss:
```diff
@@ -302,9 +302,23 @@
 
   .#{$prefix}--snippet-button .#{$prefix}--btn--copy__feedback {
-    top: rem(25px);
+    top: rem(50.8px);
```
Where is this number coming from?

joshblack commented on packages/components/src/components/code-snippet/_code-snippet.scss:
```diff
@@ -302,9 +302,23 @@
+  .#{$prefix}--snippet--multi
+    .#{$prefix}--snippet-button
+    .#{$prefix}--btn--copy__feedback {
+    top: rem(42.8px);
```
Where is this number coming from?

joshblack reviewed (changes requested):
Any chance we could avoid the magic numbers or document them? Seems like we're setting up the placement to be very precise with the decimal pixels

asudoh committed:
fix(code-snippet): fix font family of the copy button

asudoh commented:
Fixed the tooltip offset, and added the comment how the right offset is generated. Made a change to force serif font as our type token of choice doesn't dictate the font face and code snippet UI has monospace font.

emyarod reviewed (commented):
the magic numbers in code snippet should be resolved by pending work for #4605 right?

on a side note the copy button tooltip positioning should be resolved in #4715

emyarod commented on packages/components/src/components/code-snippet/_code-snippet.scss:
```diff
@@ -302,9 +303,27 @@
     left: 50%;
     right: auto;
+
+    &:before {
```
should we use the CSS3 selector with double colon instead? same with L316

```suggestion
    &::before {
```

laurenmrice reviewed (approved):
placement of tooltip and the type looks good to me 👍

asudoh committed:
Update packages/components/src/components/code-snippet/_code-snippet.scss

Co-Authored-By: emyarod <emyarod@users.noreply.github.com>

joshblack committed:
Merge branch 'master' into copy-button-feedback-text

asudoh committed:
Merge branch 'master' into copy-button-feedback-text
